docker-compose.rollkit.yml

```diff
@@ -126,14 +127,24 @@ services:
   #     - rollkit-network
 
   evm-prover:
-    image: ghcr.io/celestiaorg/celestia-zkevm-ibc-demo/evm-prover:latest
+    build:
+      dockerfile: ./docker/evm_prover.Dockerfile
     container_name: evm-prover
+    volumes:
+      - ./testing/files:/testapp_files
+      - ./.tmp:/.tmp
     environment:
       # TENDERMINT_RPC_URL should be the SimApp which is acting as a substitute
       # for Celestia (with IBC Eurekea enabled).
-      - TENDERMINT_RPC_URL=http://simapp-validator:26657
+      - SIMAPP_RPC_URL=grpc://simapp-validator:9090
+      - INDEXER_URL=http://indexer:8080
       - RPC_URL=http://reth:8545
       - EVM_PROTO_DESCRIPTOR_PATH=proto_descriptor.bin
+      - EVM_RPC_URL=http://reth:8545
```

Comment:
[nit][flup] we should de-duplicate `RPC_URL` and `EVM_RPC_URL`. Slight preference to keep `EVM_RPC_URL` b/c it is more explicit

Reply:
Followed up in #252